src/routing/core/mod.rs

```diff
@@ -176,6 +177,37 @@ impl Core {
                 }
 
                 self.print_network_stats();
+
+                // Sending SectionKnowledge to other sections for new SAP.
+                let section_auth = self.section.proven_authority_provider();
+                for sap in self.network.all() {
+                    let chain = self
+                        .section
+                        .chain()
+                        .get_proof_chain_to_current(&sap.section_key)
```

Comment:
This would be from their section PK, right? Which might not be in our chain at all?